demo/deploy-mysql.sh

```diff
@@ -46,10 +46,8 @@ spec:
         app: mysql
     spec:
       serviceAccountName: mysql
-      nodeSelector:
-        kubernetes.io/os: linux
       containers:
-      - image: mysql:5.6
+      - image: mysql/mysql-server
```

Comment:
For the sake of not changing more than we need to, is there a 5.6 or similar tag we can use? I doubt using latest will cause any problems, just want to keep the scope of these changes narrow.

Reply:
Yeah we can do something like that.